libs/shared/components/src/auth/UserMenu/UserDelegations.tsx

```diff
@@ -62,11 +62,13 @@ export const UserDelegations = ({
     delegations.push(
       ...data.authActorDelegations
         .map((delegation) => ({
-          nationalId: delegation.from.nationalId,
-          name: delegation.from.name,
+          nationalId: delegation.from?.nationalId ?? '',
+          name: delegation.from?.name ?? '',
```

Comment:
We should think about how we should handle the delegations list in the user menu when the `from` object is not found in NationalRegistry.